waku/factory/external_config.nim

```diff
@@ -300,12 +300,13 @@ type WakuNodeConf* = object
 
     pubsubTopics* {.
       desc: "Default pubsub topic to subscribe to. Argument may be repeated.",
+      defaultValue: @[],
```

Comment:
Shall we mark this `pubsubTopic` config attribute as deprecated? In that case, we will need to keep it for two more versions.

Reply:
Yes, the idea is to deprecate it and stop creating confusion/overlap between `pubsubTopic` and `shard`.
So we will have to keep it for 2 more releases (announcing the deprecation in the release notes) and afterwards create a new PR deprecating it

Reply:
Perhaps add `Deprecated` in the `desc`?